static/examples/cstg-prebid-example/index.html

```diff
@@ -106,10 +106,9 @@
     </script>
   </head>
   <body>
-    <h1>UID2 Prebid.js CSTG Integration Example</h1>
+    <h1>UID2 Prebid.js Express Integration Example</h1>
     <p class="intro">
-      This example demonstrates how a content publisher can use the <a href="https://unifiedid.com/docs/guides/integration-prebid">Prebid.js UID2 module</a>
-      to generate client-side UID2 advertising tokens.
+      This example demonstrates how a content publisher can integrate with UID2 and Prebid.js using <a href="https://unifiedid.com/docs/guides/integration-prebid">the Express Integration Guide</a>, which includes generating UID2 tokens within the browser.
```

Comment:
Suggest slight mod so that the link text matches the link destination:
This example demonstrates how a content publisher can integrate with UID2 and Prebid.js using the <a href="https://unifiedid.com/docs/guides/integration-prebid">Prebid.js Express Integration Guide</a>, which includes generating UID2 tokens within the browser.

Reply:
fixed